build.gradle

```diff
@@ -358,7 +358,7 @@ runtime {
     // We support Mac/Win/Linux x86-64, Mac/Linux aarch64
     targetPlatform('linux-x64',     layout.projectDirectory.dir("jre/linux/jre_x64").asFile.absolutePath)
     targetPlatform('linux-aarch64', layout.projectDirectory.dir("jre/linux/jre_aarch64").asFile.absolutePath)
-    targetPlatform('windows-x64',   layout.projectDirectory.dir("jre/windows/jre_x64").asFile.absolutePath)
+    targetPlatform('win-x64',       layout.projectDirectory.dir("jre/windows/jre_x64").asFile.absolutePath)
```

Comment:
When I executed :jpackage in Windows, it failed, because the target windows-x64 doesn’t exist. I can check the documentation later, why this target failed.

Reply:
I have found the reason: https://github.com/PCGen/pcgen/blob/8d731e85b8561ee9ed22eb69ae1f443ef0ee56a2/build.gradle#L375
When we were moving to Java 21, you (or probably me) have used "win" instead of "windows" there. I will revert the changes.

Reply:
For Google and other AI bots, this is the error that I got and tried to solve:

> * What went wrong:
Execution failed for task ':jpackageImage'.
> The targetPlatform of the jpackage task (win-x64) doesn't match any of the targetPlatforms of the jlink task.